README.rst

```diff
@@ -1,81 +1,98 @@
-This package contains the Zephyr RTOS meta tool, 'west'.
+This is the Zephyr RTOS meta tool, ``west``.
 
-WARNING
--------
+Installation
+------------
 
-DO NOT INSTALL WITH "python3 setup.py install".
+Install West's bootstrapper with pip::
```

Comment:
I wonder if we should capitalize West, or leave it lowercase `west`.

Reply:
Sure, I'll lowercase it.